lib/timers.js

```diff
@@ -492,6 +496,7 @@ function unrefTimeout() {
       if (domain) domain.enter();
       threw = true;
       debug('unreftimer firing timeout');
+      first._called = true;
```

Comment:
Strictly, this one wasn't neccessary to fix the issue, but I thought it's better to track all calls if there are more edge cases.